README.md

```diff
@@ -145,12 +159,14 @@ $ station --help
 Usage: station <command> [options]
 
 Commands:
-  station                   Start Station                              [default]
-  station    metrics        Show metrics
-  station    activity       Show activity log
-  station    logs [module]  Show module logs
+  station                Start Station                                 [default]
+  station metrics        Show metrics
+  station activity       Show activity log
+  station logs [module]  Show module logs
 
 Options:
+  -l, --listen   Open HTTP API                                         [boolean]
+  -p, --port     HTTP API port                          [number] [default: 7834]
```

Comment:
These two options are relevant only to `station`; they don't apply to `station metrics` and friends, right?

I also don't see the `--port` argument defined anywhere in `bin/station.js`.

Reply:
Oh, that's outdated. Will fix!